samples/snippets/src/main/java/com/example/logging/ListLogs.java

```diff
@@ -28,35 +28,64 @@
 /** List logs programmatically using the Cloud Logging API. */
 public class ListLogs {
 
-  /** Expects an existing Cloud Logging log name as an argument. */
   public static void main(String... args) throws Exception {
+
+    printLogNames();
+
+    /** Expects an existing Cloud Logging log name as an argument. */
+    String logName = args[0];
+
+    printLogEntries(logName);
+  }
+
+  public static final void printLogNames() {
```

Comment:
The [samples format guidelines](https://github.com/GoogleCloudPlatform/java-docs-samples/blob/master/SAMPLE_FORMAT.md#class-structure) say one snippet per file.

Reply:
The samples are broken into ListLogEntries.java for listing log entries and ListLogs.java for listing log names.